api/forms.py

```diff
@@ -85,3 +87,30 @@ def listing_submit():
 
     response = {'status': 200}
     return response
+
+
+# Route for submitting password reset:
+@forms.route('/reset-password-submit', methods=['POST'])
```

Comment:
Should be a PUT request since it is modifying the database

Reply:
Agreed as we are changing information already in the DB, not adding new info